### ISSUE TITLE
>>> from super_gradients.training import models ---> 2024-02-21 10:49:28.3284456 [W:onnxruntime:Default, onnxruntime_pybind_state.cc:1641 onnxruntime::python::CreateInferencePybindStateModule] Init provider bridge failed.

### ISSUE THREAD

**Issue by RamesChan** [Deci-AI/super-gradients#1850]
### 🐛 Describe the bug

`from super_gradients.training import models`
This indicates that something failed and caused the model to not predict the results.

My Code:
```
from super_gradients.training import models
from super_gradients.common.object_names import Models
import torch

yolo_nas_pose = models.get("yolo_nas_pose_l", pretrained_weights="coco_pose").cuda()

device = 'cuda' if torch.cuda.is_available() else 'cpu'

yolo_nas_pose.to(device).predict("3.jpg", conf=0.3).show()
```
![Screenshot 2024-02-21 105550](https://github.com/Deci-AI/super-gradients/assets/128948774/804e7d11-2938-4a0c-9177-599d794977cb)

```
>>> from super_gradients.training import models
2024-02-21 10:49:28.3284456 [W:onnxruntime:Default, onnxruntime_pybind_state.cc:1641 onnxruntime::python::CreateInferencePybindStateModule] Init provider bridge failed.
```

Even though it shows failed, the code can run until the end. But what happens is that there is no plot pose estimate in the showed image. And when writing the predict code, to make variable 'results' , the 'results' is Blank.

### Versions

>>> from super_gradients.training import models
2024-02-21 10:49:28.3284456 [W:onnxruntime:Default, onnxruntime_pybind_state.cc:1641 onnxruntime::python::CreateInferencePybindStateModule] Init provider bridge failed.
```

Even though it shows failed, the code can run until the end. But what happens is that there is no plot pose estimate in the showed image. And when writing the predict code, to make variable 'results' , the 'results' is Blank.
